omeroserver/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG OMEROSERVER_BASE_IMAGE=openmicroscopy/omero-server:latest
 FROM $OMEROSERVER_BASE_IMAGE
 
 USER root
-RUN yum install -y -q \
+RUN dnf install -y -q --nogpgcheck\
```

Comment:
As I mentioned to @sbesson, I recently ran into an issue where I couldn't get GPG checks to work at build time because the Docker version of the *host* (!) was too old. Do you still have the error that you ran into, @jburel?

Reply:
```
#6 7.835 Importing GPG key 0x442DF0F8:
#6 7.835  Userid     : "PostgreSQL RPM Building Project <pgsql-pkg-yum@postgresql.org>"
#6 7.835  Fingerprint: 68C9 E2B9 1A37 D136 FE74 D176 1F16 D2E1 442D F0F8
#6 7.835  From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-PGDG
#6 9.610 Error: Failed to download metadata for repo 'pgdg-common': repomd.xml GPG signature verification error: Bad GPG signature
```

Reply:
Note that the error is not related to ``dnf``, it is the same with ``yum``

Reply:
The error happens when I run the locally or via GHA (ubuntu-latest)

Reply:
Unsure if it's the same issue but the RPMs and GPG keys of the PostgreSQL RPM repository have been regenerated on 2024-01-03 - see https://yum.postgresql.org/news/pgdg-rpm-repo-gpg-key-update/

Reply:
I think it is related